editor.planx.uk/src/@planx/components/TextInput/model.ts

```diff
@@ -76,6 +115,7 @@ export const parseTextInput = (
   title: data?.title || "",
   description: data?.description,
   fn: data?.fn,
-  type: data?.type,
+  type: data?.type ?? TextInputType.Short,
```

Comment:
:heavy_check_mark: great solution to handling existing components with  now-retired "default"/undefined type !